src/Pages/HomePage.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import db from "../db (1).json";
+import { MdLocationCity, MdWebAsset, MdLocationOn } from "react-icons/md";
+import "../Pages/HomePage.css";
+
+export default function HomePage() {
+  const [agencies, setAgencies] = useState([]);
+
+  const fetchData = () => {
+    const response = db;
+    setAgencies(response.agencies);
+  };
+
+  useEffect(() => {
+    fetchData();
+  }, []);
+
+  return (
+    <div>
+      <div className="logo">
+        <h1>FNDR</h1>
+      </div>
+      {agencies.map((agency) => {
+        return (
+          <div className="block">
+            <div className="agency-card-container">
+              <div className="company-logo">
+                <img src={agency.eguideImageSrc} />
```

Comment:
This needs an alt. Try to check warnings before you PR

Here is some context an the alt attribute:
https://blog.hubspot.com/marketing/image-alt-text